classes/Domain/Services/ResetEmailer.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace OpenCFP\Domain\Services;
+
+class ResetEmailer
```

Comment:
Could we make this a mailing abstraction, in general? Thought would be we'd start to migrate other code to depend on this abstraction rather than explicitly using `Swift_Mailer`.

Would also mean that (unless there is specific logic going on here) this would become merely a contract that would be implemented in `OpenCFP\Infrastructure\Mail` (or similar). Check out the `IdentityProvider` and implementations in `OpenCFP\Infrastructure\Auth` for an example of this style.